server/punishments.ts

```diff
@@ -655,7 +656,7 @@ export const Punishments = new class {
 		}
 
 		for (const curUser of affected) {
-			curUser.locked = id;
+			curUser.locked = curUser.id;
```

Comment:
?

Reply:
i did not mean to leave that there... oops.
was trying something - fixed now with `IDOrMonitor`.